### ISSUE TITLE
Bulk import of metadata

### ISSUE THREAD

**Issue by dxcore35** [natelandau/obsidian-metadata#8]
Bulk import of the metadatas from CSV file:

file        | new_metadata
--------|-----------
file1.md | project
file2.md | area
file3.md | area

- this will add new YAML field "new_metadata"
- and for each file matching file name in column "file add string from column "new_metadata"

I have so many additional metadata to add via python.

**Comment by natelandau** [comment]
adding new metadata is definitely on the roadmap.  I was thinking about doing so without needing a separate CSV file.  My thoughts on the architecture was:

1. A regex for filenames
2. where to add metadata - ie. `frontmatter` or as an inline `key:value`
3. the metadata key
4. the metadata value (or values)

would this work for your use case?

**Comment by dxcore35** [comment]
For me personaly:
- I mass edit all data in excel with equations 
- the only missing thing for me right now is to import the property into relevant .md file
- all data is ready to be read just from the file

I cannot image how your architecture will help in this case.

**Comment by natelandau** [comment]
Makes sense, this script is still in the earliest stages of development.  I'll think through an architecture for CSV imports and hopefully include it soon.

really appreciate the feedback

**Comment by natelandau** [comment]
bulk edits of metadata via CSV are now available in v0.9.0.  Could use some help kicking the tires
